lib/urcli-money.js

```diff
@@ -163,6 +173,11 @@ class Report {
     }
 
     console.log(chalk.white(`Total Earned: ${currencyFormatter.format(this.totalEarned, {code: 'USD'})}`))
+
+    //Only print Daily Average if thea rgument is not 'today'
```

Comment:
Yep! I've added support to `yesterday`. It's in 927955c0d917f55b346c9681f8f325595d4e5546